pkgs/by-name/wt/wtf/package.nix

```
@@ -0,0 +1,34 @@
+{ pkgs ? import <nixpkgs> { }, lib ? pkgs.lib, fetchFromGitHub, }:
+
+pkgs.stdenv.mkDerivation rec {
+  pname = "wtf";
+  version = "0.5.7";
+
+  src = fetchFromGitHub {
+    owner = "0vercl0k";
+    repo = "wtf";
+    tag = "v${version}";
+    hash = "sha256-lfPx9RQEW457VQkDbvu/D9EFZrdNLz2ToQ9dfa7+tzY=";
+  };
+
+  nativeBuildInputs = [ pkgs.cmake pkgs.ninja pkgs.pkg-config ];
+
+  sourceRoot = "source/src";
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp wtf $out/bin/
+  '';
+
+  meta = {
+    description = "wtf is a cross-platform snapshot-based fuzzer";
```

```suggestion
    description = "Cross-platform snapshot-based fuzzer";
```
The package nane should not be part of the description